react-native/README.md

```diff
@@ -94,13 +94,15 @@ See [the objC style guide](../osx/STYLEGUIDE.md)
 ### Javascript settings
 
 ```
-npm install -g standard
-npm install -g babel-eslint
 npm install -g flow
 ```
 
 See [the JS style guide](standardjs.com)
+We're extending standard to be more strict in some cases (see our .eslintrc)
+
 Visit [standardjs.com](http://standardjs.com/#text-editor-plugins) to find plugins for your editor of choice
+We're including local plugins which requires you to run eslint locally as well (see https://github.com/eslint/eslint/issues/1238)
```

Comment:
I added a comment about con already below that?

On Thursday, November 19, 2015, Marco Munizaga <notifications@github.com>
wrote:

> In react-native/README.md
> <https://github.com/keybase/client/pull/1364#discussion_r45383121>:
>
> >  Visit [standardjs.com](http://standardjs.com/#text-editor-plugins) to find plugins for your editor of choice
> > +We're including local plugins which requires you to run eslint locally as well (see https://github.com/eslint/eslint/issues/1238)
>
> I use vim, what should I do?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1364/files#r45383121>.
>